File: Travelo.Model/Accomodation.cs
﻿using System;
using System.Collections.Generic;
using Travelo.Model;

namespace Travelo.Model
{
    public class Accomodation
    {
        public int Id { get; set; }
        public string Name { get; set; }


        public int CityId { get; set; }
        public string CityName { get; set; }
        public string Address { get; set; }
        public string PostalCode { get; set; }


        public string Description { get; set; }

        public ICollection<string> Facilities { get; set; }

        public byte[] Images { get; set; }
    }
}

File: travelo_agency/lib/models/accomodation.dart
import 'package:json_annotation/json_annotation.dart';

part 'accomodation.g.dart';

@JsonSerializable()
class Accomodation {
  int? id;
  String? name;
  int? cityId;
  String? cityName;
  String? address;
  String? postalCode;
  String? description;
  List<String>? facilities;
  String? locationMap;
  String? images;

  Accomodation();

  factory Accomodation.fromJson(Map<String, dynamic> json) =>
      _$AccomodationFromJson(json);
}
